WooCommerce/Classes/ViewModels/Order Details/Receipts/ReceiptEligibilityUseCase.swift

```diff
@@ -44,6 +44,10 @@ private extension ReceiptEligibilityUseCase {
     enum Constants {
         static let wcPluginName = "WooCommerce"
         static let wcPluginMinimumVersion = "8.7.0"
-        static let wcPluginDevVersion: [String] = ["8.6.0-dev-7625495467-gf50cc6b", "8.6.0-dev-7708067547-g87e8c5f"]
+        static let wcPluginDevVersion: [String] = ["8.7.0-dev",
+                                                   "8.6.0-dev",
+                                                   "8.5.0-beta.1",
+                                                   "8.6.0-dev-7625495467-gf50cc6b",
+                                                   "8.6.0-dev-7708067547-g87e8c5f"]
```

Comment:
Just for me to learn, "8.6.0-dev" is not enough to cover all 8.6.0 versions, we need the other explicit ones like "8.6.0-dev-7625495467-gf50cc6b" also?